### ISSUE TITLE
Search result popover sometimes not being shown on Wayland

### ISSUE THREAD

**Issue by ryonakano** [ryonakano/atlas#35]
Not always, but it's sometimes not shown with the following error message:

```
Gsk-Message: 19:53:09.229: Failed to realize renderer of type 'GskGLRenderer' for surface 'GdkWaylandPopup': Failed to create EGL display
```

- Ubuntu 22.04.2 LTS on Wayland
- Installed from the latest main (14f991be9b6ee87a208d3f62242383c3316ff047) using flatpak-builder

**Comment by ryonakano** [comment]
Setting to lower priority since the app targets elementary OS which is using X11 at the moment

**Comment by ryonakano** [comment]
Closing because I can't reproduce this anymore on Early Access 8 with Wayland session. Might be fixed by 56d2f0c9a6b8df825d39bf83372f017572ce0841
